ACE/ace/Monotonic_Time_Policy.h

```diff
@@ -16,6 +16,12 @@
 # pragma once
 #endif /* ACE_LACKS_PRAGMA_ONCE */
 
+#if defined (ACE_WIN32) || defined (ACE_HAS_CLOCK_GETTIME_MONOTONIC)
```

Comment:
Shouldn't the part for ACE_WIN32 be in the config-win32-common.h and the other part in another config file?

Reply:
I'm not sure about doing that since this is specific to `Monotonic_Time_Policy`. I can you insist that's right.

Reply:
This block could be moved to config-macros.h (preferably) or alternatively to Monotonic_Time_Policy.inl.

Reply:
Putting it in `Monotonic_Time_Policy.inl` is the opposite of what I want because the entire point of this is to expose the support status. However since you both seem to agree it shouldn't go here, I will break it up like @jwillemsen  suggested and put 

```
#define ACE_HAS_MONOTONIC_TIME_POLICY
```

in `config-win32-common.h` and

```
#if  ACE_HAS_CLOCK_GETTIME_MONOTONIC
#  define ACE_HAS_MONOTONIC_TIME_POLICY
#endif
```

in `config-macros.h` like @mcorino suggested.